docs/getting-started.md

```diff
@@ -143,15 +143,24 @@ Install KubeVirt, Flannel and Cluster Network Operator:
 kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml
 
 # Check for latest version
-RELEASE=v0.16.0
-kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt.yaml
 
-kubectl apply -f https://raw.githubusercontent.com/kubevirt/ovs-cni/master/examples/kubernetes-ovs-cni.yml
+# On other OS you might need to define it like
+export KUBEVIRT_VERSION="v0.18.0"
```

Comment:
current version is >> v0.30.0 - would you like to update it to more advanced one ?
Is it based on the recent work to install kubevirt on top of cento-8 ?